src/Slider/Slider.js

```diff
@@ -1,4 +1,4 @@
-import React, {Component, PropTypes} from 'react';
+import React from 'react';
```

Comment:
Could you revert this change? See https://github.com/callemall/material-ui/pull/4058 for more details.